typescript-test-samples/launchdarkly-lambda-dynamodb/cdk/resources/lambdas/getFlagsStarter/index.ts

```diff
@@ -0,0 +1,43 @@
+import { Handler, APIGatewayEvent } from "aws-lambda";
+// The LaunchDarkly Node Server SDK
+// This needs to be npm installed or added via the Lambda Layer
+import LaunchDarkly from "launchdarkly-node-server-sdk";
+
+export const handler: Handler = async (event: APIGatewayEvent) => {
+  // Initialize the client with your LaunchDarkly SDK key
+  // This needs to be added to the environment variables of your Lambda function
```

Comment:
This should be set by the CDK deployment. Please add a placeholder there that users can update before they deploy (looks like all functions need it). Please add instructions in the README to update it.